README.md

```diff
@@ -19,6 +19,19 @@ You can find my collection of configuration files for VSCode at
 https://github.com/openSUSE/suse-vscode-doc -- feel free to add your suggestions
 via GitHub pull requests or issues.
 
+## Extensions
+
+Many tasks can be automatized by using *extensions*. They are available from
+VSCode marketplaces:
+* for VSCode: https://marketplace.visualstudio.com/VSCode
+* for VSCodium: https://open-vsx.org/
+
+The documentation team delivers the following extension for VSCode:
+* DocBook Snippets
+* SUSE Vale rules
+
+TIP: you can install them from VSCode by pressing CTRL+SHIFT+X and typing the
```

Comment:
Is it only for VSCode? Or does it also works on VSCodium? If both works, maybe use a neutral word:

```suggestion
TIP: you can install them from the editor by pressing CTRL+SHIFT+X and typing the
```